minigalaxy/ui/library.py

```diff
@@ -89,23 +88,18 @@ def __create_gametiles(self) -> None:
         games_with_removed_tiles = []
         for child in self.flowbox.get_children():
             tile = child.get_children()[0]
-            if tile.current_state == tile.state.INSTALLED:
-                if not tile.game.image_url:
-                    games_with_removed_tiles.append(copy.deepcopy(tile.game))
-                    self.flowbox.remove(tile)
```

Comment:
Could you explain why this is not longer needed? It has been a long time since I wrote this code.

Reply:
OK. So now this 2 if statements are combined in one line (92). This reduces number of indentations and make code more readable. flowbox.remove() was executed here just to execute flowbox.add few lines later in __add_gametile(). Now games_with_removed_tiles (well, when I think about it now, it doesn't seems to be propitiate name any more) do not go to __add_gametile() any more to avoid just simple tile removal and addition. This is part of avoiding double game reinitialization.